spec/models/news_tematica/news_tematica_spec.rb

```diff
@@ -13,27 +13,27 @@
     end
 
     it "debe devolver 'hace una semana' si la última news de esta temática es antigua" do
-      create(:news_tematica, tematica: tematica, fecha_hasta: Time.parse("Feb 03 2013"))
-      create(:news_tematica, tematica: tematica, fecha_hasta: Time.parse("Feb 01 2013"))
+      create(:news_tematica, suscribible: tematica, fecha_hasta: Time.parse("Feb 03 2013"))
+      create(:news_tematica, suscribible: tematica, fecha_hasta: Time.parse("Feb 01 2013"))
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.